src/components/utils/index.js

```diff
@@ -164,3 +164,11 @@ export const capitalise = (string) => {
 
   return string
 }
+
+/**
+ * Generate Base64 string from raw string to use as key in iterator
+ * It's necessary to encode and unescape here is to work with non-Latin characters
+ * See more: https://stackoverflow.com/a/26603875
```

Comment:
There's a note on this StackOverflow solution about mobile Safari:
> Note: if you need to get this to work in mobile-safari, you might need to strip all the white-space from the base64 data...

Has it been checked if this is it no longer required (since this post was from a few years back anyway)?

Reply:
Thanks for pointing it out @kopijunkie, it works normally on mobile Safari.

Reply:
Cool. LGTM then! 👍